_docs/tasks/istio-auth.md

```diff
@@ -58,40 +60,38 @@ boundary. This will offer strong security for Istio CA.
 * Services in the same Kubernetes cluster but different namespaces are able to talk to each other through Istio Auth
 without extra trust setup.
 
-#### Deplying CA
+#### Deploying CA
 
 The following command creates  namespace *istio-system* and deploys CA into the namespace:
 
 ```bash
-kubectl apply -f ./kubernetes/istio-auth/istio-cluster-ca.yaml
+kubectl apply -f templates/istio-auth/istio-cluster-ca.yaml
 ```
 
-#### <a name="istioconfig"></a>Enabling Istio Auth in Istio config
+#### Deploying other Istio components
 
-The following command uncomments the line *authPolicy: MUTUAL_TLS* in the file *kubernetes/istio-X.yaml*,
-and backs up the original file as *istio-X.yaml.bak*
-(*X* corresponds to the Kubernetes server version, choose "15" or "16").
+The following command will enable mTLS, and the services will use the per-cluster CA deployed in the last step.
 
 ```bash
-sed "s/# authPolicy: MUTUAL_TLS/authPolicy: MUTUAL_TLS/" ./kubernetes/istio-X.yaml > ./kubernetes/istio-auth-X.yaml
+kubectl apply -f templates/istio-auth/isio-auth-per-cluster-ca.yaml
 ```
 
 #### Deploying other services
 
-Follow [the general Istio installation guide](./installing-istio.html),
-and **choose "If you would like to enable Istio Auth" in step 3**.
+Follow [the general Istio installation guide](./installing-istio.html) from step 5.
 
-## Disabling Istio Auth
+## <a name="disableauth"></a>Disabling Istio Auth
```

Comment:
Why do you need this <a> link? Every header in markdown gets an id auto-assigned. In this case, this heading has id of disabling-istio-auth.